src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js

```diff
@@ -29,10 +30,14 @@ export const buildCpsAssetPageATIParams = (
   const chapter1 = isValidPage ? getChapter1(page) : false;
   const ldpThingIds = getThingAttributes('thingId', pageData);
   const ldpThingLabels = getThingAttributes('thingLabel', pageData);
+  const isLegacyMediaAssetPage = url => url.split('/').length > 7;
```

Comment:
this code is no longer just MAP specific so would be nice if could rename this variable (just this one - there's probably a lot more pre-existing stuff being mis-named in here, but would be nice to not add to the problem, at least 😄 )